staging/src/k8s.io/apiserver/pkg/server/storage/resource_config.go

```diff
@@ -147,22 +121,6 @@ func (o *ResourceConfig) ResourceEnabled(resource schema.GroupVersionResource) b
 	return true
 }
 
-func (o *ResourceConfig) AllResourcesForVersionEnabled(version schema.GroupVersion) bool {
-	versionOverride, versionExists := o.GroupVersionResourceConfigs[version]
-	if !versionExists {
-		return false
-	}
-	if !versionOverride.Enable {
-		return false
-	}
-
-	if len(versionOverride.EnabledResources) == 0 && len(versionOverride.DisabledResources) == 0 {
-		return true
-	}
-
-	return false
-}
-
 func (o *ResourceConfig) AnyResourcesForVersionEnabled(version schema.GroupVersion) bool {
```

Comment:
doesn't this function go away too since we're now only testing at the version level?

Reply:
Yes, to clean thoroughly , this function should be removed and add a new one called VersionEnabled to check if a version is enabled.

Reply:
This changes so many files, I am not sure whether worth it before. 
Now since you support, I can do this completely.